src/FakeItEasy/ReturnValueConfigurationExtensions.cs

```diff
@@ -342,6 +323,5 @@ public static void ReturnsNextFromSequence<T>(this IReturnValueConfiguration<Tas
 
             configuration.ReturnsLazily(x => queue.Dequeue()).NumberOfTimes(queue.Count);
         }
-#endif
     }
 }

```

Comment:
I don't even care if this file compiles anymore, I love the new version so much more.